src/styles/utils/variables.scss

```
@@ -0,0 +1,9 @@
+$nav-color: #00acdc;
```

According to other variables
```suggestion
$color-nav: #00acdc;
```